modules/space/rendering/renderabletravelspeed.cpp

```diff
@@ -150,7 +163,7 @@ RenderableTravelSpeed::RenderableTravelSpeed(const ghoul::Dictionary& dictionary
         if (SceneGraphNode* n = sceneGraphNode(_targetName);  n) {
             _targetNode = n;
             _targetPosition = _targetNode->worldPosition();
-            _lightTravelTime =
+            _travelTime =
                 glm::distance(_targetPosition, _sourcePosition) / _travelSpeed;
```

Comment:
This looks like it could be combined into one line?

Reply:
Sharp 👀 you got there 
![image](https://github.com/OpenSpace/OpenSpace/assets/8808894/646d64e0-6bec-424e-864d-6545907ec8fb)